spec/bulk_update_file_spec.rb

```diff
@@ -78,10 +78,13 @@ def call(**options)
     expect { call }.to output("[1/1] alphagov/foo ❌ repo doesn't exist (or we don't have permission)\n").to_stdout
   end
 
-  it "skips repos where the branch already exists" do
```

Comment:
I'd delete this test from the "Checks if PR exists" commit, as I imagine this test was failing at that point in the commit history. And then you end up effectively changing the test in the "Raises PRs if the branch already exists" commit, so there's little value in keeping the original around.